identity-issuance/src/main/java/com/android/identity/issuance/funke/FunkeIssuingAuthorityState.kt

```diff
@@ -262,17 +263,20 @@ class FunkeIssuingAuthorityState(
         )?.let {
             WalletApplicationCapabilities.fromCbor(it.toByteArray())
         } ?: throw IllegalStateException("WalletApplicationCapabilities not found")
+        val useGermanId = metadata.authorizationServers.isNotEmpty()
+                && metadata.authorizationServers[0].useGermanEId
         return FunkeProofingState(
-            credentialIssuerUri = credentialIssuerUri,
-            clientId = clientId,
-            issuanceClientId = issuanceClientId,
-            documentId = documentId,
-            proofingInfo = proofingInfo,
-            applicationCapabilities = applicationCapabilities,
-            // Don't show TOS when using browser API
-            tosAcknowleged = !authorizationMetadata.useGermanEId,
-            openid4VpRequest = openid4VpRequest
-        )
+                clientId = clientId,
```

Comment:
Looks like this is now indented 8 from the previous line instead of 4? Is that what Android Studio or a linter defaulted this to? Otherwise it seems like this should be 4.

Reply:
Just a result of sloppy restructuring of the code.